### ISSUE TITLE
weeder seems to misparse build-depends

### ISSUE THREAD

**Issue by sjakobi** [ndmitchell/weeder#38]
weeder-1.0.3 reports the following for `threepenny-gui`:

```
== Section library ==
Redundant build-depends entry
* aeson(
```

I suspect that this is due to the version bounds on that dependency:

```
                    ,aeson                  (>= 0.7 && < 0.10) || == 0.11.* || (>= 1.0 && < 1.4)
```

**Comment by ndmitchell** [comment]
Thanks for the info, should be easy to fix - will do so later tonight.

**Comment by ndmitchell** [comment]
Thanks! Yep, I've improved the package name parser, so now it should work robustly. I'll make a new release after the CI agrees with me.

**Comment by ndmitchell** [comment]
Released as 1.0.4.

**Comment by sjakobi** [comment]
Cheers! :)
